src/robot/parsing/model/statements.py

```diff
@@ -974,6 +1008,74 @@ def from_params(cls, values=(), indent=FOUR_SPACES, separator=FOUR_SPACES, eol=E
         return cls(tokens)
 
 
+@Statement.register
+class Continue(Statement):
+    type = Token.CONTINUE
+
+    @classmethod
+    def from_params(cls, indent=FOUR_SPACES, eol=EOL):
+        return cls([
+            Token(Token.SEPARATOR, indent),
+            Token(Token.CONTINUE),
+            Token(Token.EOL, eol)
+        ])
+
+    def validate(self):
+        if self.get_tokens(Token.ARGUMENT):
+            self.errors += ('CONTINUE does not accept arguments.',)
+
+
+@Statement.register
+class Break(Statement):
+    type = Token.BREAK
+
+    @classmethod
+    def from_params(cls, indent=FOUR_SPACES, eol=EOL):
+        return cls([
+            Token(Token.SEPARATOR, indent),
+            Token(Token.BREAK),
+            Token(Token.EOL, eol)
+        ])
+
+    def validate(self):
+        if self.get_tokens(Token.ARGUMENT):
+            self.errors += ('BREAK does not accept arguments.',)
+
+
+@Statement.register
+class Continue(Statement):
+    type = Token.CONTINUE
+
+    @classmethod
+    def from_params(cls, indent=FOUR_SPACES, eol=EOL):
+        return cls([
+            Token(Token.SEPARATOR, indent),
+            Token(Token.CONTINUE),
+            Token(Token.EOL, eol)
+        ])
+
+    def validate(self):
+        if self.get_tokens(Token.ARGUMENT):
+            self.errors += ('CONTINUE does not accept arguments.',)
+
+
+@Statement.register
+class Break(Statement):
+    type = Token.BREAK
+
+    @classmethod
+    def from_params(cls, indent=FOUR_SPACES, eol=EOL):
+        return cls([
+            Token(Token.SEPARATOR, indent),
+            Token(Token.BREAK),
+            Token(Token.EOL, eol)
+        ])
+
+    def validate(self):
+        if self.get_tokens(Token.ARGUMENT):
+            self.errors += ('BREAK does not accept arguments.',)
```

Comment:
No need to to have `Break` and `Continue` three times!

Not sure what would be the best place for them in this rather long module. Probably after `Return`.

Reply:
Removed.